railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -49,7 +77,27 @@ def destroy
     @<%= orm_instance.destroy %>
     redirect_to <%= index_helper %>_url, notice: <%= "'#{human_name} was successfully destroyed.'" %>
   end
+  <% if collection_routing? %>
+  # Collection routes
+  # PATCH <%= route_url %>/1..10
+  def update_many
+    <%= orm_class.find(class_name, "#{singular_table_name}_ids") %>.each do |<%= singular_table_name %>|
+      <%= singular_table_name %>.update(params[:<%= plural_table_name %>][:"#{<%= singular_table_name %>.id}"])
```

Comment:
Should this line use `<%= "#{plural_table_name}_params" %>` rather than `params[:<%= plural_table_name %>]`?

In other words, the generated method now looks like:

```ruby
  def update_many
    Post.find(post_ids).each do |post|
      post.update(params[:posts][:"#{post.id}"])
    end
    redirect_to @post, notice: 'Posts were successfully updated.'
  end
```

but maybe it could look like

```ruby
  def update_many
    Post.find(post_ids).each do |post|
      post.update(posts_params[:"#{post.id}"])
    end
    redirect_to @post, notice: 'Posts were successfully updated.'
  end
```

What do you think?